svelte-kit-scss/src/lib/components/Profile/OrganizationProfile/OrganizationProfile.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { beforeEach, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import OrganizationProfile from './OrganizationProfile.svelte';
+import { MOCK_GITHUB_ORGANIZATION_PROFILE_INFORMATION } from '$lib/helpers/mocks/users';
+import { remapPublicProfileInformation } from '$lib/helpers';
+
+const mockOrgProfile = remapPublicProfileInformation(MOCK_GITHUB_ORGANIZATION_PROFILE_INFORMATION);
+
+describe('OrganizationProfile', () => {
+  beforeEach(() => {
+    render(OrganizationProfile, {
+      profile: mockOrgProfile,
+    });
+  });
+
+  it.each([
```

Comment:
I think it might help to split these into individual tests since we may want to have different validations on the different fields. Eg. it displays a correctly formatted email or the number of followers is actually a number.